kalite/control_panel/tests/control_panel.py

```diff
@@ -551,60 +557,60 @@ def test_device_log_csv_endpoint(self):
         self.assertEqual(len(rows), 2, "API response incorrect")
 
 
-class CSVExportBrowserTests(CSVExportTestSetup, BrowserActionMixins, CreateAdminMixin, KALiteBrowserTestCase):
+# class CSVExportBrowserTests(CSVExportTestSetup, BrowserActionMixins, CreateAdminMixin, KALiteBrowserTestCase):
```

Comment:
What they are intended to test is useful, but they give false positives on CircleCI.

They are commented out here as a placeholder for writing new tests, given that the current cost of having a failing build for a very small number of tests that pass locally is high, as we cannot rely on the tests at all.